lib/sequin/consumers/source_table.ex

```diff
@@ -126,40 +183,53 @@ defmodule Sequin.Consumers.SourceTable.IntegerValue do
   end
 end
 
-defmodule Sequin.Consumers.SourceTable.FloatValue do
+defmodule Sequin.Consumers.SourceTable.BooleanValue do
   @moduledoc false
   use Ecto.Schema
 
   import Ecto.Changeset
 
   @primary_key false
   embedded_schema do
-    field :value, :float
+    field :value, :boolean
   end
 
   def changeset(struct, params) do
-    struct
-    |> cast(params, [:value])
-    |> validate_required([:value])
+    case validate_boolean_format(params) do
+      {:ok, validated_params} ->
+        struct
+        |> cast(validated_params, [:value])
+        |> validate_required([:value])
+
+      {:error, error} ->
+        struct
+        |> cast(%{}, [])
+        |> add_error(:value, error)
+    end
   end
-end
-
-defmodule Sequin.Consumers.SourceTable.BooleanValue do
-  @moduledoc false
-  use Ecto.Schema
 
-  import Ecto.Changeset
+  @error_msg ~s(must be either "true" or "false")
```

Comment:
```suggestion
  @error_msg ~s(must be either `true` or `false`)
```

? make it a little clearer they don't need quotes?